cypress/support/index.ts

```diff
@@ -17,8 +17,12 @@
 import "cypress-wait-until";
 import "./commands";
 
-Cypress.on("uncaught:exception", err => {
-  console.debug("Uncaught exception", err);
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
I think we should probably get rid of this "uncaught:exception" handler; it makes it much harder to see where the error actually occurs and any uncaught exceptions we should probably handle...